src/components/worldclock.js

```diff
@@ -0,0 +1,10 @@
+import Clock from './clock';
+
+export default function WorldClock({clockData}) {
+  
+  return (
+    <div>
+      {clockData.map((timezoneValue, i) => <Clock timeZone={timezoneValue} key={i}/> )}
```

Comment:
LOVE this, so far you are the only one who has used `.map` to render the different clocks, great job mate